### ISSUE TITLE
System.Diagnostics.Tests.ProcessThreadTests.TestStartTimeProperty failing

### ISSUE THREAD

**Issue by krwq** [dotnet/corefx#5439]
Not sure if this is intermittent issue or consistent repro - will update whenever I hit this again.

Here is the stack:
```
     System.Diagnostics.Tests.ProcessThreadTests.TestStartTimeProperty [FAIL]
        Assert+WrapperXunitException : File path: e:\oss\corefx\src\System.Diagnostics.Process\tests\ProcessThreadTests.cs. Line: 85
        ---- Assert.InRange() Failure
        Range:  (635884096988417476 - 635884096988617476)
        Actual: 635884096988643468
        Stack Trace:
           e:\oss\corefx\src\Common\tests\System\Diagnostics\AssertWithCallerAttributes.cs(582,0): at Assert.WrapException(Exception e, String callerFilePath, Int32 callerLineNumber)
           e:\oss\corefx\src\Common\tests\System\Diagnostics\AssertWithCallerAttributes.cs(225,0): at Assert.InRange[T](T actual, T low, T high, String path, Int32 line)
           e:\oss\corefx\src\System.Diagnostics.Process\tests\ProcessThreadTests.cs(85,0): at System.Diagnostics.Tests.ProcessThreadTests.TestStartTimeProperty()
           ----- Inner Stack Trace -----
           e:\oss\corefx\src\Common\tests\System\Diagnostics\AssertWithCallerAttributes.cs(224,0): at Assert.InRange[T](T actual, T low, T high, String path, Int32 line)
```

OS: Win 10
note: machine I used for this build is pretty slow (no SSD)

Might be related:
https://github.com/dotnet/corefx/pull/2822
https://github.com/dotnet/corefx/issues/2613

**Comment by krwq** [comment]
Setting milestone to RC2 as it needs to be triaged and it might be a product bug

**Comment by Priya91** [comment]
fixed with #5752
